src/9FC031D0.c

```diff
@@ -89,7 +88,10 @@ s32 func_9FC03410(void* randomOut, s32 nWords) {
     SHA1Context sha1ctx;
     u8 randomBytes[0x200];
     u8 sp270[125][0x14];
-    u32 spC38[5];
+    union {
+        u32 words[5];
+        u8 bytes[0x14];
+    } spC38;
```

Comment:
I wonder if for stuff like this it would be better to do this instead

```suggestion
    union {
        u32 words[5];
        u8 bytes[5 * sizeof(u32)];
    } spC38;
```
Maybe even use a macro for the `5`? idk

Reply:
I think 0x14 will probably have a define as the SHA1 hash length, but I feel it may be a bit out of scope to add that right now?